pkg/controllers/dynakube/version/reconciler_test.go

```diff
@@ -107,21 +107,25 @@ func TestReconcile(t *testing.T) {
 			timeProvider:   timeProvider,
 			dtClient:       mockClient,
 		}
-		err := versionReconciler.Reconcile(ctx)
+		err := versionReconciler.ReconcileCM(ctx)
+		require.NoError(t, err)
+		err = versionReconciler.ReconcileAG(ctx)
+		require.NoError(t, err)
+		err = versionReconciler.ReconcileOA(ctx)
 		require.NoError(t, err)
 		assertStatusBasedOnTenantRegistry(t, dynakube.DefaultActiveGateImage(), testActiveGateImage.Tag, dkStatus.ActiveGate.VersionStatus)
 		assertStatusBasedOnTenantRegistry(t, dynakube.DefaultOneAgentImage(), testOneAgentImage.Tag, dkStatus.OneAgent.VersionStatus)
 		assert.Equal(t, latestAgentVersion, dkStatus.CodeModules.VersionStatus.Version)
 
 		// no change if probe not old enough
 		previousProbe := *dkStatus.CodeModules.VersionStatus.LastProbeTimestamp
-		err = versionReconciler.Reconcile(ctx)
+		err = versionReconciler.ReconcileCM(ctx)
 		require.NoError(t, err)
 		assert.Equal(t, previousProbe, *dkStatus.CodeModules.VersionStatus.LastProbeTimestamp)
 
 		// change if probe old enough
 		changeTime(timeProvider, 15*time.Minute+1*time.Second)
-		err = versionReconciler.Reconcile(ctx)
+		err = versionReconciler.ReconcileCM(ctx)
```

Comment:
same here out of consistency we might want to write out the name instead of abbreviating

Reply:
done